libs/components/angular-tree-component/src/lib/modules/angular-tree/angular-tree-node.component.ts

```diff
@@ -319,4 +348,14 @@ export class SkyAngularTreeNodeComponent implements AfterViewInit, OnInit {
       return false;
     }
   }
+
+  #updateDefaultContextMenuAriaLabel(): void {
+    this.#contextMenuResourceUnsubscribe.next();
```

Comment:
this could be simplified a tiny bit--we could use the new `DestroyRef`. That would get rid of the need for `ngOnDestroy`, and then you can just store the resource subscription as a variable and unsubscribe if it exists before subscribing again.